index.html

```diff
@@ -12,7 +12,7 @@
         <h1>Hello World</h1>
     </div>
 
-    <script type="module" src="./App.js"></script>
+    <script type="module" src="./src/App.js"></script>
```

Comment:
observe the type="module" and is required as
imports and exports cannot be performed in an js file